nav2_bringup/params/nav2_params.yaml

```diff
@@ -198,6 +198,13 @@ local_costmap:
       resolution: 0.05
       robot_radius: 0.22
       plugins: ["voxel_layer", "inflation_layer"]
+      filters: ["keepout_filter"]
```

Comment:
Remove, we're not going to change the default behavior for users mid-distribution (even if they are newly available)